CHANGELOG.md

```diff
@@ -7,9 +7,8 @@ All notable changes to this extension will be documented in this file.
 ### Added
 
 - The Flink Artifacts / Flink UDFs view is now searchable.
-
-### Changed
```

Comment:
Was this header supposed to be removed?

Reply:
Prob not. In fact, all these could more easily be considered changes, not additions. Sigh.